catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -96,9 +99,31 @@ const SearchLayout: FunctionComponent = ({ children }) => {
     }
   }, [currentSearchCategory]);
 
+  const updateUrl = (form: HTMLFormElement) => {
+    const urlQuery = formDataAsUrlQuery(form);
+    console.log(urlQuery);
+    router.push({ pathname: router.pathname, query: urlQuery });
+  };
+
   return (
     <CataloguePageLayout {...pageLayoutMetadata}>
       <div className="container">
+        <form
+          role="search"
+          id="searchPageForm"
+          onSubmit={event => {
+            event.preventDefault();
+
+            trackEvent({
```

Comment:
Could we list all the added tracking events somewhere (maybe start a ticket) so we can discuss them with Tacey in our tracking catchups? It's good to add them, just want to double check how we want to structure the info within!